components/Common/ProgressionSideBar/ProgressionSideBarIcon/index.tsx

```diff
@@ -13,4 +13,4 @@ const ProgressionSidebarIcon: FC<SVGAttributes<SVGSVGElement>> = props => (
   </svg>
 );
 
-export default ProgressionSidebarIcon;
+export default ProgressionSideBarIcon;
```

Comment:
this seems like an arbitrary change

- it's lowercase in the figma
- it's a compound single word according to https://www.merriam-webster.com/dictionary/sidebar
- The entirety of the `Sidebar` component family is lowercased
- the rename didn't carry through to the usage within components/Common/ProgressionSideBar/ProgressionSideBarItem/index.tsx

`import ProgressionSidebarIcon`

I'm like [5 outta 10](https://capwatkins.com/blog/the-sliding-scale-of-giving-a-fuck) on changing it, but am concerned its not a complete rename. Note the filename vs the variable names.

![image](https://github.com/nodejs/nodejs.org/assets/298435/de065f2f-9de1-453f-baf8-b0af20d0af18)

Reply:
Hmm, right it should be Sidebar, will do the change.